airflow/api_connexion/parameters.py

```diff
@@ -48,6 +45,27 @@ def format_datetime(value: str):
         )
 
 
+def check_limit(value: int):
+    """
+    This checks the limit passed to view and raises BadRequest if
+    limit exceed user configured value
+    """
+    max_val = int(conf.get("api", "maximum_page_limit"))
+    spec = {'default': 100, 'maximum': max_val,
+            'minimum': 1, 'type': 'integer'}
+    if value > max_val:
+        # This message should not be formatted. If formatted please
+        # run tests
+        message = ("{value} is greater than the maximum of {max_val}\n"
+                   "\n"
+                   "Failed validating 'maximum' in schema:\n"
+                   "    {spec}\n"
+                   "\nOn instance:\n    {value}").format(value=value,
+                                                         max_val=max_val, spec=spec)
```

Comment:
```suggestion
        message = (f"{value} is greater than the maximum of {max_val}")
```
This error is not due to schema validation. We have security on another layer.